field.go

```diff
@@ -90,24 +97,53 @@ done:
 // parseFieldAfterType expects:
 // fieldName "=" fieldNumber [ "[" fieldOptions "]" ] ";
 func parseFieldAfterType(f *Field, p *Parser, parent Visitee) error {
-	pos, tok, lit := p.next()
-	if tok != tIDENT {
-		if !isKeyword(tok) {
-			return p.unexpected(lit, "field identifier", f)
+	expectedToken := tIDENT
+	expected := "field identifier"
+
+	for expectedToken != -1 {
+		pos, tok, lit := p.next()
+		if tok == tCOMMENT {
+			c := newComment(pos, lit)
+			if f.InlineComment == nil {
+				f.InlineComment = c
+			} else {
+				f.InlineComment.Merge(c)
+			}
+			continue
+		}
+		if tok != expectedToken {
+			return p.unexpected(lit, expected, f)
+		} else {
```

Comment:
`else` is implied here since the above branch would return, if you wanted to flatten the rest of this loop.

Reply:
thx for your comments, will process them now